tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -51,167 +51,177 @@ class TestTiffImagingInterface(ImagingExtractorInterfaceTestMixin):
     save_directory = OUTPUT_PATH
 
 
-@parameterized_class(
-    [
-        {
-            "interface_kwargs": dict(
-                file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"),
-                channel_name="Channel 1",
-            ),
-            "photon_series_name": "TwoPhotonSeriesChannel1",
-            "imaging_plane_name": "ImagingPlaneChannel1",
-        },
-        {
-            "interface_kwargs": dict(
-                file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"),
-                channel_name="Channel 4",
-            ),
-            "photon_series_name": "TwoPhotonSeriesChannel4",
-            "imaging_plane_name": "ImagingPlaneChannel4",
-        },
-    ],
-)
-class TestScanImageImagingInterfaceMultiPlaneCase(ScanImageMultiPlaneImagingInterfaceMixin):
+class TestScanImageImagingInterfaceMultiPlaneChannel1(ScanImageMultiPlaneImagingInterfaceMixin):
     data_interface_cls = ScanImageImagingInterface
     interface_kwargs = dict(
-        file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"),
+        file_paths=[OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"],
         channel_name="Channel 1",
+        interleave_slice_samples=True,
     )
     save_directory = OUTPUT_PATH
 
     photon_series_name = "TwoPhotonSeriesChannel1"
     imaging_plane_name = "ImagingPlaneChannel1"
     expected_two_photon_series_data_shape = (6, 256, 528, 2)
-    expected_rate = 29.1248
-    expected_starting_time = 0.0
+    expected_rate = None  # This is interleaved data so the timestamps are written
```

Comment:
Ah this comment is very useful, I would include this in the interface docstring for `interleave_slice_samples`